src/scripts.js

```diff
@@ -65,6 +65,7 @@ function dropYear(dates) {
   return reformattedDates
 }
 $(document).ready(function () {
+  console.log('ready!')
```

Comment:
I think we should try and keep our main clear of console.logs whenever we are pushing up to it.